src/test/groovy/com/mechanitis/demo/spock/OrderingServiceTest.groovy

```diff
@@ -0,0 +1,100 @@
+package com.mechanitis.demo.spock
+
+import com.synacy.gradprogram.spock.exercise.*
+import spock.lang.Specification
+
+class OrderingServiceTest extends Specification {
+
+    OrderingService service
+    Cart cart
+
+    def items = [
+            new Item("Chicken Cordon", 25, ItemType.FOOD),
+            new Item("Trouser", 1500, ItemType.CLOTHING),
+            new Item("Smart-TV", 3500, ItemType.APPLIANCE),
+            new Item("IPhone14Pro", 62000, ItemType.GADGET),
+            new Item("Pancit Malabon", 110, ItemType.FOOD)
+    ]
+
+    def setup() {
+        service = new OrderingService()
+    }
+
+    def "Validate the Items if there have food Order on the Cart"() {
+        when:
+        boolean validateCartItem = service.cartContainsFoodItem(cart = new Cart(UUID.randomUUID(), items))
+
+        then:
+        validateCartItem
```

Comment:
This is okay but it would be easier to read if we specify what we are expecting like this:
```
true == containsFoodItem
```

This should apply to the rest of the boolean assertions.

Also we need to have a test when the scenario is the Cart does not have any food item.